src/incubator/ChipsInput/index.tsx

```diff
@@ -142,6 +142,7 @@ const ChipsInput = forwardRef((props: ChipsInputProps, refToForward: React.Ref<a
       ref={fieldRef}
       leadingAccessory={chipList}
       blurOnSubmit={false}
+      style={{flexGrow: undefined}}
```

Comment:
Don't pass inline styles
It creates a new object ref every render which triggers subsequent re-renders for inner components that depends on this prop